lib/caps/DG1/iHD

```diff
@@ -36,7 +36,11 @@ caps = dict(
       fmts      = ["NV12", "AYUV"],
       features  = dict(scc = True, msp = True),
     ),
-    hevc_10 = dict(maxres = res8k , fmts = ["P010", "Y410"]),
+    hevc_10 = dict(
+      maxres = res8k ,
+      fmts = ["P010", "Y410"],
+      features  = dict(m10sp = True)
```

Comment:
10 bit is already implied in `hevc_10`.  Please just use `msp = True`

Reply:
ok

Reply:
updated, thanks